assets/js/blocks/product-tag/editor.scss

```diff
@@ -0,0 +1,9 @@
+.wc-block-product-tag__selection {
+	width: 100%;
+}
+.wc-block-product-tag__cancel-button.is-tertiary {
```

Comment:
This selector doesn't seem to match any button. I guess we might need to change the class in `block.js` or change the selector here.